package-lock.json

```diff
@@ -0,0 +1,2795 @@
+{
```

Comment:
Similar to the new CSS file - does this need to be here, or has it been copied by accident?

Reply:
Removed file as it is not in line with what's on master, not sure why it was generated.